drivers/ethernet/phy/phy_mii.c

```diff
@@ -30,10 +32,12 @@ struct phy_mii_dev_data {
 	const struct device *dev;
 	phy_callback_t cb;
 	void *cb_data;
-	struct k_work_delayable monitor_work;
 	struct phy_link_state state;
 	struct k_sem sem;
+#if !ONLY_FIXED_LINK
```

Comment:
wouldn't it be better to define the macro as reverse instead of putting ! on all uses of it. Also maybe should use UTIL_NOT

Reply:
Sure